### ISSUE TITLE
[bzip2] Build error

### ISSUE THREAD

**Issue by markdhenning** [microsoft/vcpkg#32107]
[issue_body.md](https://github.com/microsoft/vcpkg/files/11793098/issue_body.md)

This error is _NOT_ vcpkg's fault, but it is a problem for which we hopefully can find a resolution.

What happens is: cl.exe writes an exe file.  **Corporate Controlled** Enterprise Virus Scanner begins to scan new exe file. MT.exe tries to update the manifest, but cannot open the exe file R/W because the scanning service has the file locked.

I am working with my cybersecurity department to get a generalized scanning exception for C:\src\vcpkg for my team, but a universal fix for this kind of problem would be to add a way to cause MT.exe to delay opening the file for a configurable amount of milliseconds to allow the scan to complete.  One option would be an environment variable, e.g.  MT_EXECUTE_DELAY=1000, along with a visual studio property or vcpkg manifest option to set the same for vcpkg manifest mode would resolve the problem for most.

**Comment by autoantwort** [comment]
https://github.com/microsoft/vcpkg/blob/223b8d5695bd0615e1eeba393c64770f6faf20e7/scripts/cmake/vcpkg_execute_build_process.cmake#L89-L116
We only need a way to sleep cross platform

**Comment by dg0yt** [comment]
> It could be easily integrated here

No, this is pointless. It is too far away from the many spots where the error can occur.
Apart from `mt` itself being fixed, the immediate mitigation would be a wrapper for `mt` which does nothing but re-run the same command line with some delay.
